File: 867-transpose-matrix/867-transpose-matrix.cpp
class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>>& vec) {
        vector<vector<int>>ans;
        for(int i=0;i<vec[0].size();i++)
        {
            vector<int>v;
            for(int j=0;j<vec.size();j++)
            {
                v.push_back(vec[j][i]);
            }
            ans.push_back(v);
        }
        return ans;
    }
};

File: 867-transpose-matrix/867-transpose-matrix.py
class Solution:
    def transpose(self, vec: List[List[int]]) -> List[List[int]]:
        ans=[];
        for i in range(0,len(vec[0])):
            v=[]
            for j in range (0,len(vec)):
                v.append(vec[j][i]);         
            ans.append(v);
        return ans;
        